.github/workflows/e2e-tests-ci.yml

```diff
@@ -104,9 +104,25 @@ jobs:
           echo "Cypress run completed with $CYPRESS_FAILURES failures"
           [ "$CYPRESS_FAILURES" = "0" ]
 
-  update-final-status:
+  update-failure-final-status:
+    runs-on: ubuntu-22.04
+    if: failure() || cancelled()
+    needs:
+      - smoketests
+    steps:
+      - uses: mattermost/actions/delivery/update-commit-status@main
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        with:
+          repository_full_name: ${{ github.repository }}
+          commit_sha: ${{ inputs.commit_sha || github.sha }}
+          context: ci/e2e-smoketests
```

Comment:
@mvitale1989 Should we go with `E2E Tests/smoketests` to follow the trend ? 
In the future if we add more we can just change the after `/` part ? 

cc: @isacikgoz @saturninoabril

Reply:
Yes, sounds good to me, i'll push the change in a bit